rules/common_settings.bzl

```diff
@@ -123,6 +123,24 @@ string_list_flag = rule(
     doc = "A string list-typed build setting that can be set on the command line",
 )
 
+def _flag_impl(ctx):
```

Comment:
Nit: rename to `_repeatable_string_flag_impl` - there are a lot of flag rules in this file!